daemon/network.go

```diff
@@ -315,9 +315,21 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		driver = c.Config().DefaultDriver
 	}
 
+	networkOptions := create.Options
+	for _, d := range [...]string{driver, "*"} {
+		if defaultOpts, ok := daemon.configStore.DefaultNetworkOpts[d]; ok {
+			for k, v := range defaultOpts {
+				logrus.Debugf("Applying default option for network: %v=%v", k, v)
```

Comment:
```suggestion
				logrus.Debugf("Applying default option %v=%v to network %v", k, v, id)
```
(There's probably a structured log field for network IDs but I can't think of what the field name might be.)